src/services/exportInfoMap.ts

```diff
@@ -88,7 +88,8 @@ namespace ts {
                 //    get a better name.
                 const importedName = exportKind === ExportKind.Named || isExternalModuleSymbol(namedSymbol)
                     ? unescapeLeadingUnderscores(symbolTableKey)
-                    : getNameForExportedSymbol(namedSymbol, scriptTarget);
+                    : getNameForExportedSymbol(namedSymbol, /*scriptTarget*/ undefined);

```

Comment:
I decided to remove `scriptTarget` here because

- its only purpose is to allow inclusion of non-ASCII characters in filenames to become part of the generated identifier name if you’re in a newer version of ECMAScript where that’s allowed
- I’m pretty sure the Project system wasn’t taking that into account in its invalidation of this cache

So I would rather just generate ASCII-only identifiers for unnamed default exports than have to worry about that.